ethcore/src/block_queue.rs

```diff
@@ -38,6 +38,8 @@ pub struct BlockQueueInfo {
 	pub verified_queue_size: usize,
 	/// Number of blocks being verified
 	pub verifying_queue_size: usize,
+	/// Indicates queue is empty
+	pub empty: bool
```

Comment:
is this really a necessary optimisation? (why not `fn empty(&self) -> bool { verification.verified.is_empty() && verification.unverified.is_empty() && verification.verifying.is_empty() }`?